app/views/pages/accessibility.html.erb

```diff
@@ -3,37 +3,77 @@
+      <p class="govuk-body">
+        The Becoming a Teacher team (DfE) is committed to making Find postgraduate teacher training,
+        in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2)
+        Accessibility Regulations 2018.
+      </p>
 
-      <p class="govuk-body">This website is partially compliant with the <%= link_to "Web Content Accessibility Guidelines (WCAG) version 2.1", "https://www.w3.org/TR/WCAG21/", class: "govuk-link" %> AA standard, due to the non-compliances listed below.</p>
+      <p class="govuk-body">
+        Find postgraduate teacher training is fully compliant with the
+        <%= link_to "Web Content Accessibility Guidelines (WCAG) version 2.1", "https://www.w3.org/TR/WCAG21/", class: "govuk-link" %>
+        AA standard.
+      </p>
 
-      <p class="govuk-body">The service was designed using the <%= link_to "Web Content Accessibility Guidelines (WCAG) version 2.1", "https://design-system.service.gov.uk/accessibility", class: "govuk-link" %>, which conforms to the WCAG 2.1 AA standard. To ensure the service fully meets the WCAG 2.1 AA standard, we are currently arranging an accessibility audit. We will update this page following the audit, and publish details of any accessibility issues identified.</p>
+      <h3 class="govuk-heading-m">
+        How we tested this service
+      </h3>
+
+      <p class="govuk-body">
+        This service was last tested on 16 December 2019. The test was carried out by the
+        <%= link_to "Digital Accessibility Centre (DAC)", "https://digitalaccessibilitycentre.org", class: "govuk-link" %>,
+        using both automated testing tools and a manual user testing team. An accessibility
+        report was issued on 14 January 2020. All issues raised in the report have been resolved.
+      </p>
 
       <h3 class="govuk-heading-m">Reporting accessibility problems with Find postgraduate teacher training</h3>
 
-      <p class="govuk-body">If you think we’re not meeting accessibility requirements, or you’d like to access any information in a different format (such as accessible PDF, large print, easy read, audio recording or braille), contact <%= mail_to "becomingateacher@digital.education.gov.uk", "becomingateacher@digital.education.gov.uk", class: "govuk-link" %>. Include 'Accessibility issues' in the subject line of your email.</p>
+      <p class="govuk-body">
+        If you think we’re not meeting accessibility requirements, or you’d like to access
+        any information in a different format (such as accessible PDF, large print, easy read,
+        audio recording or braille), contact
+        <%= mail_to "becomingateacher@digital.education.gov.uk", "becomingateacher@digital.education.gov.uk", class: "govuk-link" %>.
+        Include 'Accessibility issues' in the subject line of your email.
```

Comment:
should we do the subject line `Accessibility issues` as well in the mailto?

Reply:
That's a good idea, We might want to still include the copy but I'll check with Paul G

Reply:
We will remove this line and add the copy to the subject line for the user

Reply:
Even better, we should use `bat_contact_mail_to` helper method to generate this link.